src/dvc_objects/fs/implementations/gdrive.py

```diff
@@ -51,172 +51,75 @@ def __init__(self, **config):
         if not opts["host"]:
             raise ConfigError("Empty GDrive URL '{}'.".format(config["url"]))
 
-        self._bucket = opts["host"]
-        self._path = opts["path"].lstrip("/")
-        self._trash_only = config.get("gdrive_trash_only")
-        self._use_service_account = config.get("gdrive_use_service_account")
-        self._service_account_user_email = config.get(
-            "gdrive_service_account_user_email"
+        self._path = opts["host"] + opts["path"]
+
+        client_id = config.get(
+            "gdrive_client_id", self.DEFAULT_GDRIVE_CLIENT_ID
         )
-        self._service_account_json_file_path = config.get(
-            "gdrive_service_account_json_file_path"
+        client_secret = config.get(
+            "gdrive_client_secret", self.DEFAULT_GDRIVE_CLIENT_SECRET
         )
-        self._client_id = config.get("gdrive_client_id")
-        self._client_secret = config.get("gdrive_client_secret")
-        self._validate_config()
-
-        self._creds_path = config.get("gdrive_user_credentials_file")
-        tmp_dir = config.get("gdrive_credentials_tmp_dir")
-        if not self._creds_path and tmp_dir:
-            self._creds_path = os.path.join(
-                tmp_dir, self.DEFAULT_USER_CREDENTIALS_FILE
-            )
-
-    @classmethod
-    def _strip_protocol(cls, path):
-        from fsspec.utils import infer_storage_options
-
-        opts = infer_storage_options(path)
-
-        return "{host}{path}".format(**opts)
+        use_service_account = config.get("gdrive_use_service_account")
+        client_user_email = config.get("gdrive_service_account_user_email")
+        profile = config.get("profile")
```

Comment:
@shcheklein I specifically used `profile` because `gdrive_*` are all so out-of-place and unlike other options. We need consistency one way or another and I just went with the easiest one.

Reply:
For the recird: `profile` is used in many other remotes.

Reply:
yep I know where it comes from, it's still very bad to my mind

I think we should rename them later all at once (if we decide to do this), one param right now won't play a role, but looks like a bug or sloppiness

Reply:
This is arguable both ways :D I just did what I would do with any new filesystem - reuse existing options. I also don't like the inconsistency of `gdrive_` options. Considering that `profile` is already there and costs us nothing - let's use it for now and get back to this discussion in the future.